flask_2/templates/hello.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1 style="size=35px; color='green'">Hello {{ name }}!</h1>
+<form action="/exit/">
```

Comment:
И тут тоже, но метод указать нужно
```suggestion
<form method="post">
```